odl/discr/discr_mappings.py

```diff
@@ -23,20 +23,18 @@
 
 # Imports for common Python 2/3 codebase
 from __future__ import print_function, division, absolute_import
-
 from future import standard_library
 standard_library.install_aliases()
 from builtins import str, super, zip
 
-# External imports
 from itertools import product
 import numpy as np
 
-# ODL imports
 from odl.operator.operator import Operator
 from odl.discr.partition import RectPartition
 from odl.space.base_ntuples import NtuplesBase, FnBase
 from odl.space.fspace import FunctionSet, FunctionSpace
+from odl.util.testutils import run_doctests
```

Comment:
That's where all the pylint complaints come from :-)

Now we don't have to obey a linter, and I agree that the additional import is somewhat clumsy. I'll move the import.